src/gateway/how-to/request-transformations.md

```diff
@@ -0,0 +1,115 @@
+request verifying the transformation process.
+
+### Prerequisites
+
+The guide assumes the following:
+
+* You have a {{site.base_gateway}} available for testing and is reachable on `localhost`. 
+If you would like help running a local gateway using Docker, see this companion 
+[{{site.base_gateway}} in minutes](/gateway/{{page.kong_version}}/how-to/kong-gateway/) guide. If your gateway is available on 
+a different host, adjust the commands in this guide appropriately.
+* You have a service named `mock` installed on your gateway. You can adjust the 
+commands to use a different service or use the [{{site.base_gateway}} in minutes](/gateway/{{page.kong_version}}/how-to/kong-gateway) guide to 
+run a gateway with a `mock` service.
+* You have [`curl`](https://curl.se/) installed on your system, which is used to send 
+requests to the gateway. Most systems come with `curl` pre-installed.
+* This guide uses the [`jq`](https://stedolan.github.io/jq/) command line JSON processing tool. While
+this tool is not necessary to complete the tasks, it's helpful for processing JSON responses from
+the gateway. If you do not have `jq` or do not wish to install it, you can modify the commands to remove
+`jq` processing.
+
+### Steps
+
+There are a large number of Kong plugins, many of which need to 
+be [custom installed](/gateway/{{page.kong_version}}/plugin-development/distribution/) 
+prior to utilization. Fortunately, Kong ships prepackaged with a number of useful plugins including
+the Request Transformer.
+
+First verify the Request Transformer plugin is available on your gateway by querying the Admin API and using `jq` to filter the response looking at the plugins available on the server.
+
+```sh
+curl -s http://localhost:8001 | \
+  jq -r '.plugins.available_on_server."request-transformer"'
+```
+
+The command output should be:
+```
+true
+```
+
+Now, assign a new instance of the Request Transformer plugin to
+the mock service by invoking the following `POST` request to the Admin API.
+In this command, the `config.add.body` value instructions the plugin to add a new
+field to the body of incoming requests before forwarding to the `mock` service.
+In this example, we are adding a field named `new-field`, and giving it a static value of
+`defaultValue`. 
+
+```sh
+curl -i -X POST http://localhost:8001/services/mock/plugins \
+  --data "name=request-transformer" \
+  --data "config.add.body=new-field:defaultValue"
+```
+
+The result of that command should be a successful `201 Created` HTTP response code with a 
+JSON body including information about the new plugin instance.
+
+The Request Transformer provides the ability to perform much more complex
+transformations, see the 
+[full documentation](/hub/kong-inc/request-transformer/) for more detail.
+
+Next, we can use the `mock` service's `/requests` endpoint to test the behavior of the plugin.
+The `/requests` API will echo back valuable information from the request we make to it including
```

Comment:
```suggestion
The `/requests` endpoint will return valuable information from the request including
```

Reply:
This is nitpicky but the "endpoint" isn't what returns the information, the upstream Service does that (at Kong we seem to tend to use the term API to mean Service which I think is a touch confusing).

Reply:
I also wanted to emphasize the term "echo" here because that is what the requests service is doing and i think that will help the reader understand what's happening.